### PR TITLE
Setup project and storybook

### PR DESCRIPTION
**Pull request by jamesricky**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "repository": "git@github.com:vivid-planet/comet-mail-components.git",
+    "scripts": {
+        "build": "tsc --project ./tsconfig.json",
+        "lint:tsc": "tsc --noEmit",
+        "start": "start-storybook -p 36245",
+        "build-storybook": "build-storybook"
+    },
+    "dependencies": {
+        "react": "^17.0.2",
+        "react-dom": "^17.0.2"
```
Comment (nsams) [inline package.json:22]: these should be peerDependencies

**Review by nsams — APPROVED** [review]

**Comment by kaufmo** [comment]
@jamesricky what is the status of this PR or Project? Do we need this?

**Comment by jamesricky** [comment]
@kaufmo The PR is ready, but the project is on hold for now while I evaluate if MJML is usable for our needs.

**Comment by johnnyomair** [comment]
@jamesricky what's the status here? 😁

**Comment by jamesricky** [comment]
> @jamesricky what's the status here? 😁

@johnnyomair ah, forgot this exists 😅 
We can delete this repo, we use MJML now and if we want some common components for mails we would probably add them to `@comet/site` or something anyway.

**Pull request closed without merging**